app/react/src/client/preview/render.tsx

```diff
@@ -4,15 +4,17 @@ import ReactDOM from 'react-dom';
 
 import { RenderContext } from './types';
 
+// will be provided by the webpack define plugin
+declare const FRAMEWORK_OPTIONS: {
```

Comment:
I took a shortcut here duplicating reactOptions type. Typings should be reworked and be under the same rootDir (./src)

Reply:
Do you need:

```
import { FRAMEWORK_OPTIONS } from 'global';
```